contracts/royalty-enforcement/RoyaltyWhitelist.sol

```diff
@@ -0,0 +1,124 @@
+//SPDX-License-Identifier: Apache 2.0
+pragma solidity ^0.8.0;
+
+// Access Control
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+// Introspection
+import "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+
+// Interfaces
+import "./IRoyaltyWhitelist.sol";
+
+/*
+    RoyaltyWhitelist is an implementation of a whitelist registry, storing addresses and bytecode
+    which are allowed to be approved operators and execute transfers of interfacing tokens. The registry
```

Comment:
`interfacing tokens` is a bit unclear IMO. Could we mention something like NFT contracts or ERC721 contracts instead?